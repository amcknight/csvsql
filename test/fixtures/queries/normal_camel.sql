SELECT FirstName, LastName
FROM test/fixtures/tables/people_camel
WHERE Age > 30
