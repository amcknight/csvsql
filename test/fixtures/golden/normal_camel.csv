FirstName, LastName
Andrew, McKnight
Tom, Ash
Cayley, Carlson
